persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -140,21 +146,21 @@ CASE
     WHEN ((SELECT id FROM SelfUser LIMIT 1) LIKE (Conversation.creator_id || '@%')) THEN 1
     ELSE 0
     END AS isCreator,
-Conversation.last_notified_message_date AS lastNotifiedMessageDate,
+Conversation.last_notified_instant AS lastNotifiedMessageInstant,
 memberRole. role AS selfRole,
 Conversation.protocol,
 Conversation.mls_cipher_suite,
 Conversation.mls_epoch,
 Conversation.mls_group_id,
-Conversation.mls_last_keying_material_update,
+Conversation.mls_last_keying_material_update_instant,
 Conversation.mls_group_state,
 Conversation.access_list,
 Conversation.access_role_list,
 Conversation.team_id,
 Conversation.mls_proposal_timer,
 Conversation.muted_time,
 Conversation.creator_id,
-Conversation.last_modified_date,
+Conversation.last_modified_instant,
```

Comment:
tbh, I found the `_date` more indicates the purpose of the field and not the type! So do we need to rename all of them with `_instant` ending? I thin `_date` would be OK!

Reply:
I tend to agree with @mchenani here, with keeping `_date`. Using a consistent suffix is nice though.

Reply:
Although it works in the DB layer, what about the Entity models? 

`ConversationEntity.lastModifiedDate` still exists (but it's deprecated now), same for other date fields 🤔 

And then it becomes a bit weird, mapping the `last_modified_date` column to something else than the `lastModifiedDate` field...

I'm trying to come up with a solution here but I can't see it yet.

Reply:
This is a temporary problem though I assume, and the usage of deprecated functions will be removed in a follow up PR.

Maybe something like this:
PR 1. introduce `lastModifiedInstant` and deprecate `lastModifiedDate` (this PR). 
PR 2. remove usage of deprecate functions
PR 3. rename `lastModifiedInstant` to `lastModifiedDate`

Reply:
I took another look at the usages of the deprecated stuff. And there aren't many.

Replacing them with the new ones actually simplifies some stuff and reduces modifications 😅 

So I just did all the 3 steps in a single commit.